lib/src/widgets/article_list_screen/article_list_screen.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:saraka/constants.dart';
+import 'package:provider/provider.dart';
+import '../wave_background.dart';
+import '../../blocs/article_list_bloc.dart';
```

Comment:
`../../xxx` should be above than `../xxx`.

Reply:
I separated the task. I will consider it in Widget part.